packages/widgets/src/commandpalette.ts

```diff
@@ -973,6 +980,41 @@ export namespace CommandPalette {
       return kb ? CommandRegistry.formatKeystroke(kb.keys) : null;
     }
 
+    /**
+     * @returns The aria label content to add to the shortcut node.
+     */
+    formatItemAria(data: IItemRenderData): h.Child {
+      const keyToText: { [key: string]: string } = {
+        ']': 'Closing bracket',
+        '[': 'Opening bracket',
+        ',': 'Comma',
+        '.': 'Full stop',
+        "'": 'Single quote',
+        '-': 'Hyphen-minus'
+      };
+
+      let kbText = data.item.keyBinding;
+      let result = kbText ? CommandRegistry.formatKeystroke(kbText.keys) : null;
+
+      let punctuationRegex = /\p{P}/u;
+      let punctuations = result?.match(punctuationRegex);
+      if (!punctuations) {
+        return [];
+      }
+      for (const punctuation of punctuations) {
+        if (result != null && Object.keys(keyToText).includes(punctuation)) {
+          const individualKeys = result.split('+');
+          let index = individualKeys.indexOf(punctuation);
+          if (index != -1) {
+            individualKeys[index] = keyToText[punctuation];
+          }
+          const textShortcut = individualKeys.join('+');
+          return textShortcut;
+        }
+      }
+      return kbText ? CommandRegistry.formatKeystroke(kbText.keys) : null;
```

Comment:
Should this logic be moved to `formatKeystroke`, or maybe even better should `CommandRegistry` have an extra function called `CommandRegistry.formatKeystrokeAriaLabel`? All the renderer would do would be calling such method? Maybe translated `keyToText` should be passed to `CommandRegistry` options to avoid redefining/passing it around in all places we will need it?